CONTRIBUTING.md

```diff
@@ -112,16 +116,13 @@ In these files(eg. basemodui_czech.txt) the untranslated strings are indented by
 ### Workshop
 Create a copy of the English file and rename it to your language suffix (eg. workshop_tags_schinese.json). Translate the contents of the file. In JSON files, only translate text on the right side of the colon `:`.
 
-### Item Schema
-Duplicate lines with an `_english` suffix and rename them according to your language. Keep each block of translations in alphabetical order by language name. If there are words between `%percent_signs%`, leave them as-is (they are replaced with numbers in the game later). Don't change any lines that aren't suffixed with a language name.
-
 ## Changing English strings
 If you change an english string in a way that does not require editing other languages (such as fixing a typo that doesn't change the meaning), you'll need to also change the `[english]` copy of the string in each of the other language files too (for positive ci-checks later on). The sync-tool will do the hard work for you. Start it after you've made your change in the english file. 
```

Comment:
You can use `utils\translation-sync-tool.exe -only-update` (maybe I should add some batch files to make this easier for people who don't use the command line) to update English strings without resetting any of the other languages, for stuff like minor punctuation and spelling fixes.

Reply:
I'll make an separate section for the tool. As it's practical in some edgecases but isn't needed for entry level stuff and included in the workflow anyway.

Reply:
Also, as far as I understand it's available as an go script too (for automation and linux users). Am I right?

Reply:
The exe is just a compiled version of the Go source code that's in the same folder.